server/src/main/java/org/spine3/server/entity/DefaultEntityStorageConverter.java

```diff
@@ -60,12 +60,11 @@ private DefaultEntityStorageConverter(Repository<I, E> repository, FieldMask fie
     protected Tuple<I> doForward(E entity) {
```

Comment:
Usage of `Tuple` with no documentations may lead to <s>brain damage</s> unexpected consequences.